pkg/metrics/metrics.go

```diff
@@ -101,6 +116,29 @@ var (
 		Help:      "Number of policies currently loaded",
 	})
 
+	// PolicyRegenerationCount is the total number of policies successfully
```

Comment:
What exactly constitutes a "policy"?

Reply:
Not sure what you are asking but In current context /code `PolicyRegenerationCount` is basically total number of times `regeneratePolicy` is computing a policy for an endpoint successfully i.e when `isPolicyComp` is set to true and no error is being returned. What do you think is unclear ? I can add more comments/make changes.